tensorizer/tensorizer.py

```diff
@@ -856,13 +856,6 @@ def df_main(args: argparse.Namespace) -> None:
         args.input_directory, use_auth_token=hf_api_token
     )
 
-    cudadev = torch.cuda.current_device()
-    gb_gpu = int(
```

Comment:
You should borrow my GPU memory usage function from `finetuner.py` as it accounts for no-GPU cases rather than just tossing out this tracking.

Although, hmm, there'd be no GPU memory to measure? Might be useful for other points where we'd want it though.

Reply:
Yeah, I think it would be useful in the validation step